diablo/externals/kaltura.py

```diff
@@ -337,37 +372,34 @@ def _create_media_template(
     ):
         entitled_users = []
         if publish_type == 'kaltura_my_media':
-            entitled_users = ','.join([instructor['email'] for instructor in instructors])
+            email_addresses = [instructor['email'] for instructor in instructors]
+            entitled_users = ','.join(to_normalized_set(email_addresses))
 
-        media_entry = KalturaMediaEntry(
+        base_entry = KalturaBaseEntry(
```

Comment:
`KalturaMediaEntry` is a subclass of `KalturaBaseEntry` and we want the latter. It serves as the primary-key that links a schedule to its categories and collaborators.